lib/wda/webdriveragent.js

```diff
@@ -96,22 +96,64 @@ class WebDriverAgent {
     }
   }
 
+  /**
+   * Return boolean if WDA is running or not
+   * @return {boolean} True if WDA is running
+   * @throws {Error} If there was invalid response code or body
+   */
   async isRunning () {
+    return !!(await this.getStatus());
+  }
+
+  /**
+   * Return current running WDA's bundle id
+   * @return {?string} bundle id
+   * @throws {Error} If there was invalid response code or body
+   */
+  async getRunningBundleId () {
+    const status = await this.getStatus();
+    if (util.hasValue(status)) {
+      return status.currentApp.bundleID;
+    }
+    return status;
+  }
+
+  /**
+   * Return current running WDA's bundle id
+   * @return {?object} State Object
+   * @throws {Error} If there was invalid response code or body
+   */
+  async getStatus () {
     const noSessionProxy = new NoSessionProxy({
       server: this.url.hostname,
       port: this.url.port,
       base: '',
       timeout: 3000,
     });
     try {
-      const status = await noSessionProxy.command('/status', 'GET');
-      if (!status) {
+      //{
+      //   "state": "success",
+      //   "os": {
+      //     "name": "iOS",
+      //     "version": "11.4",
+      //     "sdkVersion": "11.3"
+      //   },
+      //   "ios": {
+      //     "simulatorVersion": "11.4",
+      //     "ip": "172.254.99.34"
+      //   },
+      //   "build": {
+      //     "time": "Jun 24 2018 17:08:21"
+      //   }
+      // }
+      const response = await noSessionProxy.command('/status', 'GET');
```

Comment:
I ran this method. I found the response has no bundleId..

Reply:
https://github.com/appium/appium-xcuitest-driver/blob/9dbbeb9f7a819abba76d8ce18928127d5308c02b/lib/driver.js#L758 also try to get `currentApp.bundleID` as same. But the method isn't called.

Reply:
Obtain `NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];` and return the identiifer on WDA